pkg/controller/endpointslicemirroring/endpointslicemirroring_controller.go

```diff
@@ -347,11 +347,7 @@ func (c *Controller) queueEndpoints(obj interface{}) {
 // This does not ensure that a corresponding Service exists with a nil selector.
 // That check should be performed separately.
 func (c *Controller) shouldMirror(endpoints *v1.Endpoints) bool {
-	if endpoints == nil || skipMirror(endpoints.Labels) || hasLeaderElection(endpoints.Annotations) {
-		return false
-	}
-
-	return true
+	return endpoints != nil && !skipMirror(endpoints.Labels) && !hasLeaderElection(endpoints.Annotations)
```

Comment:
I personally feel the original code is easier to process, which has 3 less negations.

Reply:
Yeah, this fixes static check according to [https://staticcheck.io/docs/checks#S1008](url). Actually, the original code is more readable and easier to process. I will revert the modification in the second file or both of them, what do you think about? @caesarxuchao